module/Application/view/partial/main-nav.phtml

```diff
@@ -1,17 +1,95 @@
+<?php
+use Zend\Session\Container as SessionContainer;
+
+$session = new SessionContainer('lang');
+$lang = $session->lang; ?>
```

Comment:
It is better to use the translate plugin to get the language.
`$this->plugin('translate')->getTranslator()->getLocale();`